config/agencies.yml

```diff
@@ -288,3 +288,111 @@
   name: Vietnam Education Foundation
 21:
   name: 'State, local governments or tribes'
```

Comment:
should we deprecate this one or rename it to "other local or tribal government" ?

Reply:
Good call, I thought about it and then forgot after all the typing 😄. I think we should rename since we don't really have a mechanism to "deprecate" or mark an agency as inactive, so I think I'll update it to "Other local or tribal government" (although most local governments should fall under their respective states). Thanks!

Reply:
Other local things could be like city or county? maybe? 🤷

Reply:
Well if the goal of the Agency is to aggregate UUIDs within a given scope I think the state seems like the appropriate level for "things that aren't federal", IMO.